### PR TITLE
add publisher config

### PR DESCRIPTION
**Pull request by dacolgit**
fix deactivate checkbox
shorten security profile in endpoint page

### CODE REVIEW

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor [hunk 1]
```diff
@@ -333,40 +368,64 @@
...
+            await UnPublish(endpointId, node);
+        }
+    }
+
+    /// <summary>
+    /// Publish a node
+    /// </summary>
+    /// <param name="endpointId"></param>
+    /// <param name="node"></param>
+    private async Task Publish(string endpointId, ListNode node)
```
Comment (marcschier) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:392]: Naming: PublishAsync or PublishNodeAsync

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor [hunk 2]
```diff
@@ -333,40 +368,64 @@
...
+            node.Publishing = false;
+        }
+    }
+
+    /// <summary>
+    /// UnPublish a node
+    /// </summary>
+    /// <param name="endpointId"></param>
+    /// <param name="node"></param>
+    private async Task UnPublish(string endpointId, ListNode node)
```
Comment (marcschier) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:422]: Same as above - UnpublishAsync or so

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor [hunk 3]
```diff
@@ -151,15 +153,35 @@
...
+                        @if (node.Publishing == true && node.PublishedItem != null)
+                        {
+                            <label>Sampling:</label><a href="javascript: void(0)" @onclick="@(() => OpenDrawer(node))">
+                                @(node.PublishedItem.SamplingInterval?.TotalMilliseconds.ToString() ?? "1000") ms<br />
+                            </a>
+                            <label>Publishing: </label><a href="javascript: void(0)" @onclick="@(() => OpenDrawer(node))">
+                                @(node.PublishedItem.PublishingInterval?.TotalMilliseconds.ToString() ?? "1000") ms<br />
+                            </a>
+                            <label>HeartBeat: </label><a href="jpavascript: void(0)" @onclick="@(() => OpenDrawer(node))">
+                                @(node.PublishedItem.HeartbeatInterval == null ? "-" : node.PublishedItem.HeartbeatInterval?.TotalMilliseconds.ToString() + " ms")
```
Comment (cristipogacean) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:170]: I think the heartbeat is expressed in seconds, at least this is the convention from the old publisher

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor (hunk 1 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:392]: changed

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor (hunk 2 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:422]: changed

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor (hunk 3 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:170]: you are right. I'll change it

**Pull request merged**